tox.ini

```diff
@@ -57,3 +58,8 @@ basepython=python3
 deps=isort==5.9.1
 changedir={toxinidir}/tests
 commands=isort --thirdparty aiohttp,connexion --check-only --diff .
+
+[testenv:mypy]
+deps=mypy==0.910
+ignore_outcome=true
+commands=mypy --exclude 'examples/' --install-types --non-interactive .
```

Comment:
I'm no expert, but using `--install-types` installs the latest versions I believe? Perhaps we should explicitly add the types packages as dependencies for this step (`types-setuptools`, `types-PyYAML`, `types-requests`) to limit the upper version as in other steps? What do you think?